app/tests/integration/nimbus/test_e2e_create_experiment.py

```diff
@@ -124,7 +124,7 @@ def test_create_new_experiment_remote_settings(selenium, base_url):
     # Check it's live
     home = HomePage(selenium, base_url).wait_for_page_to_load()
     live_experiments = home.tables[0]
-    assert "live experiments" in live_experiments.table_name.lower()
+    assert home.active_tab_text, "Live"
```

Comment:
This should check that the active tab has "live experiments" within it, currently, this assert will always pass.

See: https://www.w3schools.com/python/ref_keyword_assert.asp

Reply:
@jrbenny35 Hmm, would it always pass? I want to assert that the tab is selected (contains the `.active` class). So if "Review" was selected the `active_tab_text` would not equal `"Live"`. I changed it to this since we removed the element/text that previously displayed "live experiments"

Reply:
Edit: oh I see what you meant about the usage of `assert`. Updated that, but I'm still checking for "Live" in the tab text and since "Live experiments" no longer exists.